.github/workflows/master.yml

```diff
@@ -59,19 +58,19 @@ jobs:
         toolchain: ${{ matrix.toolchain }}
         args: --target ${{ matrix.target }} --workspace
 
-    # If musl, compile and test all excluding kafka
+    # If musl, compile and test all
     - uses: actions-rs/cargo@v1
       if: matrix.target == 'x86_64-unknown-linux-musl'
       with:
-        command: build
         toolchain: ${{ matrix.toolchain }}
-        args: --target ${{ matrix.target }} --workspace --exclude cloudevents-sdk-rdkafka
+    - run: CC=musl-gcc CXX=g++ cargo build --target ${{ matrix.target }} --all
```

Comment:
Can you use the cargo action more than manually running the command?

Reply:
the action doesnt allow setting environment vars (do tell me if you know a way)

Reply:
Using env like this https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#about-environment-variables doesn't work?

Reply:
the thing is that these values get passed into the cargo build command or something:
https://github.com/fede1024/rust-rdkafka/issues/88#issuecomment-494130230
(consider this maybe xD: https://github.com/fede1024/rust-rdkafka/issues/88#issuecomment-493685113)

Reply:
well, these are just env variables, so if github actions allows you to set them, i don't see why it should not work